Packs/ThinkstCanary/Integrations/ThinkstCanary/ThinkstCanary.py

```diff
@@ -226,9 +229,12 @@ def get_token_command():
         token_file = fileResult(name, content)
         demisto.results(token_file)
     else:
-        human_readable = tableToMarkdown('Canary Tools Tokens', res.get('token'))
+        token_data = res.get('token')
```

Comment:
if we need the `.get` over direct accessors (i.e. key may not be presend), add a dictionary as default.
if we do not, replace it with `[]` access please.